### ISSUE TITLE
Error On Logout When Auth Token Is Passed Into Render Config

### ISSUE THREAD

**Issue by meddy** [coralproject/talk#1393]
### Expected behavior
If you pass an auth token into the stream embed render function you should be able to programmatically logout afterwards.

```
const embed = Talk.render(document.getElementById('talkStreamEmbed'), { authToken });
const embed.logout();
```

### Actual behavior
When you logout the comment stream crashes with an auth error "token has been revoked."

I think the issue is [this line](https://github.com/coralproject/talk/blob/master/client/coral-framework/services/bootstrap.js#L39). If the token is manually passed in through the config argument it will be used when the auth token is returned. However, the [config reducer](https://github.com/coralproject/talk/blob/master/client/coral-framework/reducers/config.js) doesn't appear to clear the auth token when you log out. So what happens is that when `logout()` is called the auth token reference persists, and the next GraphQL query that occurs fails with an auth error, since the token has been revoked on the server from the `logout()` call.

### Steps to reproduce behavior
1. Call render on the comment embed passing in an `auth_token` config option.
2. Call `embed.logout()`

**Comment by wyattjoh** [comment]
Thanks for reporting the issue!

Tracking here: https://www.pivotaltracker.com/story/show/155451710

**Comment by alinhle** [comment]
@meddy Good job, thanks for this accurate bug report! I opened up a PR to fix this #1405
